root/unconfirmed_email_discourse.tt

```diff
@@ -0,0 +1,7 @@
+[% WRAPPER 'layout.tt' title=l("Unconfirmed Email Address") %]
```

Comment:
This page will be displayed if they created an account already, and are already logged in, but don't have a confirmed email. This shouldn't happen often except to confused people.